crypto/rsa/rsa_ameth.c

```diff
@@ -773,7 +773,9 @@ static int rsa_cms_encrypt(CMS_RecipientInfo *ri)
     EVP_PKEY_CTX *pkctx = CMS_RecipientInfo_get0_pkey_ctx(ri);
     int pad_mode = RSA_PKCS1_PADDING, rv = 0, labellen;
     unsigned char *label;
-    CMS_RecipientInfo_ktri_get0_algs(ri, NULL, NULL, &alg);
+    if(CMS_RecipientInfo_ktri_get0_algs(ri, NULL, NULL, &alg) <= 0) {
```

Comment:
For style issues, please put a space after the "if" and remove the curly braces.

Reply:
No problem.  (BTW I looked through the contribution guidelines and couldn't find anything about how to nominate a change for "trivial".)

Reply:
Put the text "CLA: trivial" into your commit message.  You can use "git commit --amend" to do that.  Then push it to github via "git push -f github"  Okay?